test/db/cmd/cmd_eval

```diff
@@ -81,6 +81,7 @@ RUN
 
 NAME=multiple assignments
 FILE==
+ARGS=-ecfg.oldshell=true
```

Comment:
Remove this and fix the CMDS.. `e scr.highlight=" scr.highlight.grep  =scr.rainbow =true"` instead of the first line should work.